server/src/index.ts

```diff
@@ -25,38 +29,43 @@ i18n.configure({
 });
 
 const server = app.listen(PORT, async () => {
-    const appUrl = environments.APP_URL;
+    let appUrl = environments.APP_URL;
     // tslint:disable-next-line:no-console
     console.log(
         'App is running at http://localhost:%d in %s mode',
         PORT,
-        environmentName,
+        environmentName
     );
 
     if (environments.IsNgRokMode()) {
-        const [err, appUrl] = await catchAsyncError(
-            environments.NGROK_URL
-                ? Promise.resolve(environments.NGROK_URL)
-                : runNgrok(PORT),
+        const [err, ngRookUrl] = await catchAsyncError(
+            environments.APP_URL
+                ? Promise.resolve(environments.APP_URL)
+                : runNgrok(PORT)
         );
         // tslint:disable-next-line:no-console
         console.log(
-            `${CONSOLE_LOG_EASE_DELIMITER} NGROK started on ngRokUrl: ${appUrl}`,
+            `${CONSOLE_LOG_EASE_DELIMITER} NGROK started on ngRokUrl: ${ngRookUrl}`
         );
+        appUrl = ngRookUrl;
     }
 
     const [err, fireBaseApp] = initFirebase(environments);
     if (err) {
         // tslint:disable-next-line:no-console
         console.log(
-            `${CONSOLE_LOG_DELIMITER}Firebase did not start. Error ${err.name}, ${err.message}. Stack: ${err.stack}`,
+            `${CONSOLE_LOG_DELIMITER}Firebase did not start. Error ${err.name}, ${err.message}. Stack: ${err.stack}`
         );
     }
 
     checkCovid19Updates();
     // tslint:disable-next-line:no-console
     console.log(`${CONSOLE_LOG_DELIMITER}Starting Telegram bot`);
     runTelegramBot(app, appUrl, environments.TELEGRAM_TOKEN);
+
+    // tslint:disable-next-line:no-console
+    console.log(`${CONSOLE_LOG_DELIMITER}Starting Viber bot`);
```

Comment:
@danbilokha Do you mind adding feature flag here?

I'm not sure if it's really needed. But I'd like to have one, since it looks like we are going to launch. Our bot responder withing single app.

What is your opinion about that?

Reply:
I didn't get it

Reply:
Та забей....